packages/fxa-settings/src/pages/ConnectAnotherDevice/en.ftl

```diff
@@ -0,0 +1,25 @@
+## Connect Another Device page
+# A user will only see this header if they are signed in. The header will be preceded by a green checkmark (rtl/ltr sensitive)
+connect-another-device-signed-in-header = You’re signed into { -brand-firefox }
+# A "success" message visible to users who verified via email
+connect-another-device-email-confirmed-banner = Email confirmed
+# A "success" message visible to users who verified via sign-in
+connect-another-device-signin-confirmed-banner = Sign-in confirmed
+# A message prompts the user to sign in to this instance of the Firefox browser so as to complete device sync. This is followed by a link labeled "Sign in"
+connect-another-device-signin-to-complete-message = Sign in to this { -brand-firefox } to complete setup
+# A link for the user to sign in to the current Firefox browser, preceded by a message prompting the user to sign in so as to complete the device sync setup
+connect-another-device-signin-link = Sign in
+# A message prompting the user to sign in via a different device than the current one so as to complete the device-syncing process
+connect-another-device-still-adding-devices-message = Still adding devices? Sign in to { -brand-firefox } on another device to complete setup
+# A message prompting the user to sign in via a different device than the current one so as to complete the device-syncing process
+connect-another-device-signin-to-complete-message = Sign in to { -brand-firefox } on another device to complete setup
```

Comment:
@millsoper - Our linter just caught this, but the ID for this string is duplicated from line 9. 
Could you make a PR to fix this?

Reply:
Definitely! I'll put it together right now! Thanks for the head's up!

Reply:
Just following up to say that this was done and merged in https://github.com/mozilla/fxa/pull/14885